src/test/java/com/conveyal/datatools/manager/controllers/api/UserControllerTest.java

```diff
@@ -113,13 +116,196 @@ public void canListFirstTenUsers() throws IOException {
         assertThat(userResponse, matchesSnapshot());
     }
 
+    /**
+     * When creating a new datatools user which matches an existing Auth0 user, make sure the permissions for the
+     * existing Auth0 users can be updated.
```

Comment:
When creating a new datatools user for an email address that already exists as an Auth0 user (for a different datatools client/application in Auth0), make sure..."

Reply:
Updated.